### PR TITLE
Fix warnings/test Admin

### PR DESCRIPTION
**Pull request by 1CIM**
### Changes: 
- fix various warnings in console
- fix failing tests
- Add a PT for general dashboard. too big fix for this pr i think

### CODE REVIEW

File: cypress/integration/adminCanEditNavigation.feature.js [hunk 1]
```diff
@@ -28,19 +28,19 @@ describe('Admin Can Use Navigation Dashboard', () => {
...
-            })
-          })
+          // beforeEach(() => {
+          //   cy.intercept('PUT', '**/app_data**', {
+          //     statusCode: 200,
+          //     body: {
+          //       message: 'Info has been updated',
+          //     },
+          //   })
+          // })
```
Comment (SevaDer14) [inline cypress/integration/adminCanEditNavigation.feature.js:43]: comments

File: cypress/integration/adminCanUseSectionsDashboard.feature.js [hunk 2]
```diff
@@ -171,7 +172,7 @@ describe('Admin Can Use Sections Dashboard', () => {
         )
       })
 
-      it('is expected to create new card', () => {
+      it.only('is expected to create new card', () => {
```
Comment (SevaDer14) [inline cypress/integration/adminCanUseSectionsDashboard.feature.js:175]: .only

**Review by SevaDer14 — CHANGES_REQUESTED** [review]

File: cypress/integration/adminCanEditNavigation.feature.js (hunk 1 above)
Comment (1CIM) [reply cypress/integration/adminCanEditNavigation.feature.js]: fixed

File: cypress/integration/adminCanUseSectionsDashboard.feature.js (hunk 2 above)
Comment (1CIM) [reply cypress/integration/adminCanUseSectionsDashboard.feature.js]: fixed

**Review by SevaDer14 — APPROVED** [review]

**Pull request merged**